src/js/views/patients/sidebar/action/action-sidebar.scss

```diff
@@ -1,3 +1,26 @@
 .action-sidebar__form {
   color: color(blue);
 }
+
+.sidebar__comment-form {
+  margin-top: 16px;
+}
+
+.sidebar__comment-label {
```

Comment:
@luketlancaster, we should sync up on this. Let me know if this solution works for you or if you already have another way of displaying the initials label.

Reply:
I could maybe see moving this to a comment module.  We don't use it elsewhere, but there's nothing about this comment form that is specifically sidebar.

Reply:
this all works for me. I haven't done anything other than the most basic "I wonder what this classname should be" thinking, so I can just make whatever you have work

Reply:
That works for me. I can make adjustments to the classnames.